principles/fp-019-term-stability.md

```diff
@@ -0,0 +1,144 @@
+
+OWL: 
+```
+<owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0001574">
+    <obo:IAO_0000111>obsolete cell lysate MHC competitive binding assay using radioactivity detection</obo:IAO_0000111>
+    <obo:IAO_0000115>Competitive inhibition of binding assay measuring MHC ligand binding by radioactivity detection using MHC derived from a cell lysate</obo:IAO_0000115>
+    <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000227"/>
+    <obo:IAO_0100001 rdf:resource="http://purl.obolibrary.org/obo/OBI_0001544"/>
+    <rdfs:label>obsolete cell lysate MHC competitive binding assay using radioactivity detection</rdfs:label>
+    <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
+</owl:Class>
+```
+
+OBO: 
+```
+[Term]
+id: OBI:0001574
+name: obsolete cell lysate MHC competitive binding assay using radioactivity detection
+def: "OBSOLETE. Competitive inhibition of binding assay measuring MHC ligand binding by radioactivity detection using MHC derived from a cell lysate." []
+relationship: IAO:0000231 IAO:0000227
+is_obsolete: true
+replaced_by: OBI:0001544
+```
+For OBO format, there are multiple alternatives:
+1) Use "property_value:" instead of "relationship:".
+1) Use a free text "comment:". 
+1) Use the annotation property label (with underscores) instead of the CURIE "IAO:0000231", and the obsolescence reason label instead of the relevant CURIE. Note that the underscore version of the property label will need to be created in the ontology:
+```
+[Typedef]
+id: has_obsolescence_reason
+name: has obsolescence reason
+xref: IAO:0000233
+is_metadata_tag: true
+```
+Then:
+```
+relationship: has_obsolescence_reason IAO:0000227 ! terms merged
+```
+Or:
+```
+property_value: has_obsolescence_reason IAO:0000227
+```
+
+Counter-example
+-------
+
+The PRO term "phosphoprotein" (PR:000037070) is defined as "A protein that includes at least one phosphorylated residue." A study finds 2000 more examples than was previously known. In this case, no new term needs to be made (nor the original deleted) because (1) the term definition did not change; and (2) the referent--proteins with a phophoresidue--did not change (that is, the newly-discovered phosphoproteins are just additional examples of that referent).
+
+Criteria for review
+-------
+
+The OBO Dashboard will show:
+- An ERROR if any obsolete term (that is, a term with an "owl:deprecated" property or "is_obsolete: true" tag) does not also have 'obsolete ' (that exact string, lowercase and space included, with no other punctuation) prepended to the label
+- An ERROR if any obsolete term (as indicated by term label or definition) lacks an "owl:deprecated" property or "is_obsolete: true" tag
+- An ERROR if an obsolete term has any associated logical axioms (including any subClassOf/is_a declarations)
```

Comment:
is "associated with" the same as "referencing"?

(best to use precise terms, we generally follow the owlapi vocabulary e.g "about" "referencing")

Reply:
Yes, that's half the intent. Revising.